robot-db/src/main/java/domain/CategoryName.java

```diff
@@ -0,0 +1,13 @@
+package domain;
+
+/**
+ * Enum with available book categories
+ */
+public enum CategoryName {
+    EDUCATION_AND_SCIENCE,
+    TRAVEL,
+    LIFESTYLE,
+    SEX,
+    MEDICINE,
+    ADVENTURE;
```

Comment:
Is the order significant? First sex, then medicine, finally adventure? Shouldn't it be the other way around?